gandi/cli/commands/dnssec.py

```diff
@@ -0,0 +1,50 @@
+""" Domain namespace commands. """
+
+import click
+
+from gandi.cli.core.cli import cli
+from gandi.cli.core.params import pass_gandi
+
+
+@cli.group(name='dnssec')
+@pass_gandi
+def dnssec(gandi):
+    """Commands related to dnssec."""
+
+
+@dnssec.command()
+@click.option('--flags', default=None, type=click.INT, help='Flags')
+@click.option('--algorithm', default=None, type=click.INT, help='Algorithm')
+@click.option('--public_key', default=None, help='Public key')
```

Comment:
Those 3 parameters seems mandatory by the APIv3.   
The first two are "enum", maybe the "enum" can be define here to ? ([see](https://github.com/Gandi/gandi.cli/blob/master/gandi/cli/core/params.py))
The public-key seems to be encoded in bases 64 might be worth saying it in the help message